packages/nhost-js/src/clients/auth.ts

```diff
@@ -0,0 +1,44 @@
+import { getHasuraAuthClient } from '@nhost/hasura-auth-js'
+
+import { getAuthUrlFromEnv, urlFromParams } from '../utils/helpers'
+
+/**
+ * Get a client for Auth
+ *
+ * @param refreshIntervalTime
+ * @param clientStorageGetter
+ * @param clientStorageSetter
+ * @param clientStorage
+ * @param clientStorageType
+ * @param autoRefreshToken
+ * @param autoSignIn
+ * @param start
+ * @param urlParams
+ * @returns
+ */
+export function getAuthClient(
```

Comment:
this function and the following are not getters as they create a client every time they are called. They should be called `createAuthClient`

Reply:
:+1: